docs/detections/visual-event-analyzer.asciidoc

```diff
@@ -40,6 +49,20 @@ You can also analyze events from <<timelines-ui,Timelines>>
 
 Inside the visual analyzer, each cube represents a process (i.e. an executable file or network event). Click and drag in timeline view to see all process relationships. 
 
+To help understand what field were used to create the process, select the **Process Tree** to view the schema that created the graphical view. The fields included are:
+
+-`SOURCE`: Can be either `endpoint` or `winlogbeat`.
+- `ID`: Event field that uniquely identifies a node.
+- `EDGE`: Event field which indicates the relationship between two nodes.
```

Comment:
Line 43: 
`You can also analyze events from <<timelines-ui,Timelines>>`

You need to add a period at the end of the sentence. This would also be better wrapped in a `TIP`.